src/examples/app.py

```
@@ -0,0 +1,21 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import appier
+
+import base
+
+
+class SimpleApp(appier.APIApp):
+
+    def __init__(self, *args, **kwargs):
+        appier.APIApp.__init__(self, name="mb", *args, **kwargs)
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)